assets/variant-list.css

```diff
@@ -0,0 +1,370 @@
+}
+
+variant-list .quantity {
+  width: calc(11rem / var(--font-body-scale) + var(--inputs-border-width) * 2);
+  min-height: calc((var(--inputs-border-width) * 2) + 3.5rem);
+}
+
+variant-list .quantity__button {
+  width: calc(3.5rem / var(--font-body-scale));
+}
+
+.variant__contents {
+  position: relative;
+}
+
+.variant__items {
+  max-height: 90rem;
+  overflow-y: auto;
+  padding-bottom: 4rem;
+}
+
+.variantlist-total {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 2rem;
+  border-top: 0.1rem solid rgba(var(--color-foreground), .08);
+}
+
+@media screen and (min-width: 750px) {
+  .variantlist-total {
+    position: sticky;
+    bottom: 0;
+    z-index: 1;
+    background-color: rgb(var(--color-background));
+  }
+}
+
+.variant-items td,
+.variant-items th {
+  padding: 0;
+  border: none;
+}
+
+.variant-items th {
+  text-align: left;
+  padding-bottom: 1.8rem;
+  opacity: 0.85;
+  font-weight: normal;
+}
+
+.variant-item__quantity-wrapper {
+  display: flex;
+}
+
+.variant-item__totals,
+.variant-item__details,
+.variant-item__price {
+  position: relative;
+}
+
+.variant-items *.right {
+  text-align: right;
+}
+
+.variant-item__image-container {
+  display: inline-flex;
+  align-items: flex-start;
+  height: 4.5rem;
+  width: 4.5rem;
+}
+
+.variant-item__media {
+  width: 4.5rem;
+  margin-right: 1.2rem;
+}
+
+.variant-item__image {
+  height: 100%;
+}
+
+@media screen and (min-width: 750px) {
+  .variant-item variant-list-remove-button {
+    visibility: hidden;
+  }
+
+  .variant-item__quantity:hover variant-list-remove-button,
+  .variant-item:has(.quantity__input:focus-visible) variant-list-remove-button {
+    visibility: visible;
+  }
+
+  .variant-item__image {
+    max-width: 100%;
+  }
+
+  .variant-item__contain {
+    display: flex;
+  }
+
+  .variant-item__discounted-prices {
+    justify-content: flex-end;
+  }
+}
+
+.variant-item__details {
+  font-size: 1.6rem;
+  line-height: calc(1 + 0.4 / var(--font-body-scale));
+}
+
+.variant-item__details>* {
+  margin: 0;
+  max-width: 30rem;
+}
+
+
+.variant-item__info {
+  position: relative;
+}
+
+.variant-item__name {
+  display: block;
+}
+
+.variant-item__discounted-prices {
+  margin-top: 0;
+  margin-bottom: 0;
+  display: flex;
+  align-items: center;
+}
+
+.variant-item__discounted-prices dd {
+  margin: 0;
+}
+
+.variant-item__discounted-prices dd:first-of-type {
+  margin-right: 1rem;
+}
+
+.variant-item__discounted-prices .variant-item__old-price {
+  font-size: 1.4rem;
+}
+
+.variant-item__old-price {
+  opacity: 0.7;
+}
+
+.variant-item__final-price {
+  font-weight: 400;
+}
+
+.product-option {
+  font-size: 1.4rem;
+  word-break: break-word;
+  line-height: calc(1 + 0.5 / var(--font-body-scale));
+}
+
+.variant-item__quantity-info {
+  display: flex;
+  align-items: center;
+  margin: 0 1.5rem 0 0;
+  min-width: 1.5rem;
+  min-height: 1.5rem;
+}
+
+.variant-item__sold-out {
+  opacity: 0.7;
+}
+
+.variant-item variant-list-remove-button {
```

Comment:
These remove buttons don't appear to be focusable via tabbing. I'm not not sure how effective (or accessible) the hide/show pattern these use is. Might be nicer if these were persistent on rows with a quantity or at least the button showed up when the whole row was hovered or contained focus.

Reply:
@sofiamatulis I don't mind this feedback with the remove buttons, especially if it's an accessibility concern. 
Let's make them persistent on rows with a quantity.

Reply:
@edmund-teh to confirm the behaviour, if there are any items in cart of that row, youll always see the trash can?